src/luascript.cpp

```diff
@@ -925,14 +925,15 @@ void LuaScriptInterface::pushCombatDamage(lua_State* L, const CombatDamage& dama
 
 void LuaScriptInterface::pushInstantSpell(lua_State* L, const InstantSpell& spell)
 {
-	lua_createtable(L, 0, 6);
+	lua_createtable(L, 0, 7);
 
 	setField(L, "name", spell.getName());
 	setField(L, "words", spell.getWords());
 	setField(L, "level", spell.getLevel());
 	setField(L, "mlevel", spell.getMagicLevel());
 	setField(L, "mana", spell.getMana());
 	setField(L, "manapercent", spell.getManaPercent());
+	setField(L, "params", spell.getHasParam());
```

Comment:
`hasParam` is a boolean type, not number. I suggest renaming the field to `hasparam` and removing `> 0` from the other commit.

Reply:
i don't agree with the rename, unless you do it in spells.xml aswell
but il fix the other commit sure

Reply:
understandable
looks good to me now